quickstart_branch.sh

```diff
@@ -28,7 +28,7 @@ while (( "$#" )); do
 done
 
 # Compute the expected composer package name of the requested branch.
-if [[ $BRANCH =~ ^[0-9]\.[0-9] ]]; then
+if [[ $BRANCH =~ ^[0-9]\. ]]; then
```

Comment:
```suggestion
if [[ $BRANCH =~ ^[0-9]\.[0-9x] ]]; then
```

Reply:
@joeparsons Just to confirm, did you mean to resolve this conversation? This is the second instance where the "x" could be added, but the change hasn't been committed yet.